Library/Homebrew/formula.rb

```diff
@@ -730,12 +735,32 @@ def bottle_prefix
     prefix+".bottle"
   end
 
-  # The directory where the formula's installation logs will be written.
+  # The directory where the formula's installation or test logs will be written.
   # @private
   def logs
     HOMEBREW_LOGS+name
   end
 
+  # The prefix, if any, to use in filenames for logging current activity
```

Comment:
Few of the method comments we have in existing code seem to be complete sentences. This seems more consistent with the existing style. (Though not many methods are have method-level comments, so it's not clear there's a consistent style.)